src/redux/actions/concepts/addBulkConcepts/index.js

```diff
@@ -50,7 +50,7 @@ export const fetchFilteredConcepts = (source = 'CIEL', query = '') => async (
     dispatch(isFetching(false));
   } catch (error) {
     dispatch(isFetching(false));
-    notify.show('an error occurred, reload your browser', 'error', 3000);
+    notify.show('An error occurred with your internet connection, please fix it and try reloading the page.', 'error', 3000);
```

Comment:
Are you intentionally duplicating this message?

Reply:
@dkayiwa on this PR I just changed the way that particular message was written. I think there should be another PR to remove the duplication that was already existing.